frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -156,7 +155,7 @@ export const pointToLayer = (feature: ICluster, latlng: LatLngExpression): Layer
  */
 export const getMarkerIcon = (feature: ICluster) => {
   const { propertyTypeId, projectWorkflow, classificationId } = feature?.properties;
-  if (projectWorkflow === Workflows.ERP) {
+  if ([Workflows.ERP, Workflows.ASSESS_EX_DISPOSAL].includes(projectWorkflow)) {
```

Comment:
ERP or exemption trigger red pins.

Reply:
Not sure this is correct, as exemption are explicitly not ERP.  I suspect this would cause confusion if these properties were displayed on the map this way.

Reply:
suggestions?

Reply:
Hmm wonder if we may need to add another pin.. but that could be a separate story

Reply:
I would lean toward these pins just remaining the normal colour.